tool/speed.cc

```diff
@@ -2072,6 +2164,32 @@ bool Speed(const std::vector<std::string> &args) {
     }
   }
 
+  if (args_map.count("-primes") != 0) {
```

Comment:
Did you consider making this a function to handle chunks and primes?

Reply:
I was thinking of refactoring the shared logic to parse a comma delimited string into a list of integers which I think both primes and chunks are doing the same thing.